python/ray/ray_constants.py

```diff
@@ -67,6 +67,9 @@ def env_bool(key, default):
 # Default number of return values for each actor method.
 DEFAULT_ACTOR_METHOD_NUM_RETURN_VALS = 1
 
+# Wait 30 seconds for client to reconnect after unexpected disconnection
+DEFAULT_CLIENT_RECONNECT_GRACE_PERIOD = 30
```

Comment:
I've seen the common practices are to do exponential retries, e.g. 0.5s then 1s then 2s then 4s, etc. Does it make sense to do that or you think this is good?

Reply:
This should be fine. On the client side, since we already know that the server exists and that we have an existing session we want to retry relatively frequently (at the moment every 5s) to make sure we reconnect back to the server before the grace period expires. Exponential backoff might push the delay up to 16s, which more than likely would ruin a chance to reconnect if the connection became available again after say 20s